packages/test/src/activities/index.ts

```diff
@@ -5,6 +5,15 @@ import { Connection } from '@temporalio/client';
 import { fakeProgress as fakeProgressInner } from './fake-progress';
 import { cancellableFetch as cancellableFetchInner } from './cancellable-fetch';
 
+// TODO: Get rid of this by providing client via activity context
+function getTestConnection(): Connection {
+  const serverURLOverride = process.env.TEMPORAL_TESTING_SERVER_URL;
+  if (serverURLOverride !== undefined && serverURLOverride.length > 0) {
+    return new Connection({ address: serverURLOverride });
+  }
+  return new Connection();
+}
```

Comment:
Not that it's even close to being critical but this would have been sufficient
```suggestion
  const address = process.env.TEMPORAL_TESTING_SERVER_URL || undefined;
  return new Connection({ address });
}
```